docs/reference/config.md

```diff
@@ -513,13 +513,30 @@ nav_order: 3
 |filename|Filename is the file to write logs to.  Backup log files will be retained in the same directory|`string`|`<nil>`
 |filesize|MaxSize is the maximum size the log file before it gets rotated|[`BytesSize`](https://pkg.go.dev/github.com/docker/go-units#BytesSize)|`<nil>`
 |forceColor|Force color to be enabled, even when a non-TTY output is detected|`boolean`|`<nil>`
+|includeCodeInfo|Enables the report caller for including the calling file and line number, and the calling function. If using text logs, it uses the logrus text format rather than the default prefix format.|`boolean`|`<nil>`
```

Comment:
Note this causes FireFly to use the logrus text formatter rather than the prefixed logger if JSON logs are not enabled.